docs/en-US/component/dialog.md

```diff
@@ -125,6 +125,7 @@ When using `modal` = false, please make sure that `append-to-body` was set to **
 | modal                      | whether a mask is displayed                                                                          | ^[boolean]                          | true    |
 | modal-class                | custom class names for mask                                                                          | ^[string]                           | —       |
 | append-to-body             | whether to append Dialog itself to body. A nested Dialog should have this attribute set to `true`    | ^[boolean]                          | false   |
+| append-to                  | which element the Dialog appends to. Will override `append-to-body`                                  | ^[string]                           | body    |
```

Comment:
Perhaps a version tag `^(2.4.3)` needs to be added here.

Reply:
Added